bitsong/assetlist.json

```diff
@@ -22,7 +27,358 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitsong/images/btsg.png",
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitsong/images/btsg.svg"
         },
+        "type_asset": "sdk.coin",
         "coingecko_id": "bitsong"
+      },
+      {
+        "description": "Adam Clay a BitSong Music FanToken",
+        "denom_units": [
+          {
+            "denom": "ft2D8E7041556CE93E1EFD66C07C45D551A6AAAE09",
+            "exponent": 0
+          },
+          {
+            "denom": "clay",
+            "exponent": 6
+          },
+          {
+            "denom": "ibc/7ABF696369EFB3387DF22B6A24204459FE5EFD010220E8E5618DC49DB877047B",
+            "exponent": 0,
+            "aliases": ["ft2D8E7041556CE93E1EFD66C07C45D551A6AAAE09"]
+          }
+        ],
+        "base": "ft2D8E7041556CE93E1EFD66C07C45D551A6AAAE09",
+        "name": "Adam Clay FanToken",
+        "display": "clay",
+        "symbol": "CLAY",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitsong/images/ft2D8E7041556CE93E1EFD66C07C45D551A6AAAE09.png"
+        }
+      },
+      {
+        "description": "Nicola Fasano a BitSong Music FanToken",
+        "denom_units": [
+          {
+            "denom": "ft25B30C386CDDEBD1413D5AE1180956AE9EB3B9F7",
+            "exponent": 0
+          },
+          {
+            "denom": "fasano",
+            "exponent": 6
+          },
+          {
+            "denom": "ibc/423967B46B1A51D78619085105B04FCFA14F7CBC0BE7539A316B2DCDFC7D8C07",
+            "exponent": 0,
+            "aliases": ["ft25B30C386CDDEBD1413D5AE1180956AE9EB3B9F7"]
+          }
+        ],
+        "base": "ft25B30C386CDDEBD1413D5AE1180956AE9EB3B9F7",
+        "name": "Nicola Fasano Fantoken",
+        "display": "fasano",
+        "symbol": "FASANO",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitsong/images/ft25B30C386CDDEBD1413D5AE1180956AE9EB3B9F7.png"
```

Comment:
curious, will all these images still look appropriate if some app were to crop them to a circle, cutting off the corners?